<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>5</width>
    <height>5</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>856</width>
      <height>120</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>856</width>
      <height>120</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_a">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>a</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_b">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>b</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_c">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>c</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_d">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>d</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pushButton_e">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>e</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="pushButton_f">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>f</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="pushButton_g">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>g</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="pushButton_h">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>h</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QPushButton" name="pushButton_i">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>i</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QPushButton" name="pushButton_j">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>j</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_k">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>k</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_l">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>l</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_m">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>m</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_n">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>n</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="pushButton_o">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>o</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="pushButton_p">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>p</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QPushButton" name="pushButton_q">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>q</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QPushButton" name="pushButton_r">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>r</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QPushButton" name="pushButton_s">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>s</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="9">
      <widget class="QPushButton" name="pushButton_t">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>t</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_u">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>u</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_v">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>v</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_w">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>w</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_x">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>x</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pushButton_y">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>y</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="pushButton_z">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>z</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QPushButton" name="pushButton_caps">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Caps</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QPushButton" name="pushButton_spc">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Space</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="8" colspan="2">
      <widget class="QPushButton" name="pushButton_bkc">
       <property name="minimumSize">
        <size>
         <width>166</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Backspace</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_1">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>2</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>3</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_4">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>4</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="pushButton_5">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>5</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="pushButton_6">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>6</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QPushButton" name="pushButton_7">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>7</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QPushButton" name="pushButton_8">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>8</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="8">
      <widget class="QPushButton" name="pushButton_9">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>9</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="9">
      <widget class="QPushButton" name="pushButton_0">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="iconSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>219</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0" colspan="2">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="password">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="pass_edit"/>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="Login">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>pushButton_o</zorder>
   <zorder>username</zorder>
   <zorder>password</zorder>
   <zorder>usr_edit</zorder>
   <zorder>pass_edit</zorder>
   <zorder>Login</zorder>
   <zorder>pushButton_a</zorder>
   <zorder>pushButton_b</zorder>
   <zorder>pushButton_c</zorder>
   <zorder>pushButton_d</zorder>
   <zorder>pushButton_e</zorder>
   <zorder>pushButton_i</zorder>
   <zorder>pushButton_f</zorder>
   <zorder>pushButton_h</zorder>
   <zorder>pushButton_g</zorder>
   <zorder>pushButton_j</zorder>
   <zorder>pushButton_p</zorder>
   <zorder>pushButton_t</zorder>
   <zorder>pushButton_n</zorder>
   <zorder>pushButton_r</zorder>
   <zorder>pushButton_k</zorder>
   <zorder>pushButton_m</zorder>
   <zorder>pushButton_l</zorder>
   <zorder>pushButton_s</zorder>
   <zorder>pushButton_q</zorder>
   <zorder>pushButton_z</zorder>
   <zorder>pushButton_bkc</zorder>
   <zorder>pushButton_x</zorder>
   <zorder>pushButton_spc</zorder>
   <zorder>pushButton_u</zorder>
   <zorder>pushButton_w</zorder>
   <zorder>pushButton_v</zorder>
   <zorder>pushButton_y</zorder>
   <zorder>pushButton_caps</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_0</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>pushButton_1</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_9</zorder>
   <zorder>pushButton_7</zorder>
   <zorder></zorder>
   <zorder>label</zorder>
   <zorder>usr_edit</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
